Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 
 # stanford dlss specific gems
 gem 'assembly-objectfile'
+gem 'cocina-models'
```

Comment:
hmm, good point.  it's a holdover from the original PR, maybe from when this wasn't implicitly included via lybercore?

i guess i'd have a slight pref for including it explicitly instead of relying on its transitive inclusion, and i have a vague recollection of that lining up with our departmental tendencies.  but not something i feel strongly about, and happy to remove if you'd prefer that.

Reply:
We don't include it in the other robot repos, so would argue for consistency.

Reply:
ah, fair enough, will do